integration-tests/salesforce/src/main/java/org/apache/camel/quarkus/component/salesforce/generated/Account.java

```diff
@@ -722,21 +729,6 @@ public void setSLAExpirationDate__c(java.time.LocalDate SLAExpirationDate__c) {
         this.SLAExpirationDate__c = SLAExpirationDate__c;
     }
 
-    @XStreamConverter(MultiSelectPicklistConverter.class)
-    private Account_MyMultiselectEnum[] MyMultiselect__c;
-
-    @JsonProperty("MyMultiselect__c")
```

Comment:
Would these field removals regress https://github.com/apache/camel-quarkus/issues/3022?

Reply:
So maybe we have something different in the settings of Salesforce account.
I need to regenerate the files with my account to get the Account class compatible with Salesforce API responses.

Reply:
this is the kind of error that I have if I run my tests with my account using actual code :(

Reply:
@jamesnetherton Can you test with your account with just the commit including the test ? If you have some time

Reply:
With your changes and the previous `Account.java` it seems to be working on my account. If I consume your changes to `Account.java` then I get the error outlined in #3022.

Reply:
@jamesnetherton Oh thanks. So let me revert those changes on `Account.java`, and  SQL query.

Reply:
> @jamesnetherton Oh thanks. So let me revert those changes on `Account.java`, and SQL query.

I mean the getbyQuery test

Reply:
@jamesnetherton It should be fine now